consensus/src/simplex/mod.rs

```diff
@@ -669,11 +664,23 @@ mod tests {
                     debug!(shutdowns = *shutdowns, elapsed = ?wait, "restarting");
                     *shutdowns += 1;
                 }
+
+                // Collect supervisors
+                supervised.lock().unwrap().push(supervisors);
             }});
 
             // Recover runtime
             (executor, runtime, _) = runtime.recover();
         }
+
+        // Check supervisors for faults activity
```

Comment:
This ensures we check all faults (before, we would only check if there was a new committer item)